src/torchmetrics/classification/accuracy.py

```diff
@@ -212,11 +213,21 @@ class MulticlassAccuracy(MulticlassStatScores):
     is_differentiable = False
     higher_is_better = True
     full_state_update: bool = False
+    plot_options = {"lower_bound": 0.0, "upper_bound": 1.0, "legend_name": "Class"}
 
     def compute(self) -> Tensor:
         tp, fp, tn, fn = self._final_state()
         return _accuracy_reduce(tp, fp, tn, fn, average=self.average, multidim_average=self.multidim_average)
 
+    def plot(self, val: Optional[Union[Tensor, Sequence[Tensor]]] = None, show: bool = False) -> _PLOT_OUT_TYPE:
```

Comment:
I think this should be a static method as all the things accessed here should actually be on a per-class and not per-instance level, no? Then you could also use plotting without instantiating. This would go for all metrics accordingly.

Reply:
That's true. Ideally, we want both options...

So I should be able to do `my_acc.plot()` as well as `MulticlassAccuracy.plot([val1, val2])`.

We could do instance-checks on `val` and allow it to also be a Metric, but then we would have to go with `my_acc.plot(my_acc)` which is also a bit weird.

Or we just keep it as is, then people would have to instantiate the metric, which  would also be fine the more I think about it :)